### PR TITLE
maiden tabar and bastokan circlet sql edits, teleport earring script submission

### PR DESCRIPTION
**Pull request by tonberrylantern**

### CODE REVIEW

File: scripts/globals/items/duchy_earring [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+-- ID: 16042
+-- Duchy Earring
+-- Enchantment: "Teleport-Duchy"
+-----------------------------------------
+require("scripts/globals/teleports");
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+function onItemCheck(target)
+return 0;
```
Comment (TeoTwawki) [inline scripts/globals/items/duchy_earring]: Use indention please. The preference is 4 spaces for one indent.

Comment (TeoTwawki) [inline scripts/globals/items/duchy_earring]: Example: https://github.com/DarkstarProject/darkstar/blob/master/scripts/globals/items/anniversary_ring.lua

**Comment by bendangelo** [comment]
Looks good. It would be nice if this can be rebased into one commit.

**Comment by xdemolish** [comment]
is it currently making you teleport while the item use animation is still playing? if it is then you should probably handle it through EFFECT_TELEPORT (see items/warp_cudgel, teleports.lua, and effects/teleport.lua)

**Comment by xdemolish** [comment]
also, for future pull requests: you don't need to make a commit for every file changed, just need to commit all related file changes in one commit

for this one though, i would prefer if you were to rebase as lauty said (see https://github.com/edx/edx-platform/wiki/How-to-Rebase-a-Pull-Request if you're not sure how to go about doing this, or just hop on irc and someone will probably help)

**Comment by teschnei** [comment]
yes, you'll need to use EFFECT_TELEPORT and teleport.lua for the teleports

also, squash those commits before it can be merged

**Comment by tonberrylantern** [comment]
I'll change the teleport scripts tonight when I have a moment and can test that they work correctly, and then I'll resubmit. 
In regards to rebasing and squashing, I find plenty on google that involves command line instructions, but I'm a bit lost as everything I've done via github has just been through forking the master and editting via the web interface here- I've not done a single thing via command line. I'll see if I can figure out how to do it. 
Anyhow, thanks for the patience, I'm still very new to github if you couldn't tell.

**Comment by TeoTwawki** [comment]
https://code.google.com/p/tortoisegit/ tortoisgit ftw! Everything else failboat of with a cargo hold full of fail sailing an ocean of fail. Double the fail when its githubs application, which is even worse than trying to make their website do it all.

**Comment by teschnei** [comment]
oh god, that explains everything

check out the repository on your local machine (using git) and you can avoid making one commit per file
the command line is this bad boy over here usually: http://git-scm.com/downloads

**Comment by xdemolish** [comment]
squash dem commits
(reopen when done)

**Pull request closed without merging**